### PR TITLE
added kanbantool detector

### PR DESCRIPTION
**Pull request by lonmarsDev**

### CODE REVIEW

File: pkg/detectors/kanbantool/kanbantool.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"kanbantool"}) + `\b([0-9A-Z]{12})\b`)
+	domainPat = regexp.MustCompile(detectors.PrefixRegex([]string{"kanbantoolDomain"}) + `\b([a-z0-9A-Z._]{4,22})\b`)
```
Comment (roxanne-tampus) [inline pkg/detectors/kanbantool/kanbantool.go:25]: suggestion: `kanbantoolDomain` is unlikely to be found, recommended to just use `kanbantool`.

Comment (lonmarsDev) [reply pkg/detectors/kanbantool/kanbantool.go:25]: if we use kanbantool on domainPat as regexString and not kanbantoolDomain, regex will also get the value of keyPat

Comment (roxanne-tampus) [inline pkg/detectors/kanbantool/kanbantool.go:25]: suggestion: After some testing, they allow a minimum of 2 characters and maybe we can go with `\b([a-z0-9A-Z._]{2,22})\b`?

Comment (lonmarsDev) [reply pkg/detectors/kanbantool/kanbantool.go]: this one already resolve.

**Review by dustin-decker — APPROVED** [review]

**Pull request merged**